### PR TITLE
minor improvement on play-from-disk-vpx example

### PR DESCRIPTION
**Pull request by jurgob**
I did those improvement to avoid other people to incur in issues like this one: 
https://github.com/webrtc-rs/examples/issues/12

### CODE REVIEW

**Comment by k0nserv** [comment]
Hey @jurgob can you fix the warnings, then I'll merge.

**Comment by jurgob** [comment]
I've fixed all the warnings inside the example I 've modified.  (just to be clear,  I think none of those were added by my fix). 

I've also done some manual tests, everything seems to still working 

Do you want me to fix also the warnings for the other examples in this PR? @k0nserv

**Comment by k0nserv** [comment]
> Do you want me to fix also the warnings for the other examples in this PR?

Hmm, the reason I said that was because CI was failing due to them. Always happy to have more fixes however, but if you do please do it in a second PR.

File: Cargo.toml [hunk 1]
```diff
@@ -21,7 +21,7 @@ env_logger = "0.9.0"
 clap = "3.0.8"
 hyper = { version = "0.14.16", features = ["full"] }
 signal = {path = "examples/signal" }
-tokio-util = "0.6.9"
+tokio-util = {version="0.6.9", features = ["full"] }
```
Comment (k0nserv) [inline Cargo.toml:24]: Why was this change required?

Comment (jurgob) [reply Cargo.toml:24]: to fix this error: 

error[E0432]: unresolved import `tokio_util::codec`
  --> examples/ice-restart/ice-restart.rs:11:17
   |
11 | use tokio_util::codec::{BytesCodec, FramedRead};
   |                 ^^^^^ could not find `codec` in `tokio_util`

Comment (jurgob) [reply Cargo.toml:24]: my rational in this PR was: I m gonna fix all the errors and the warnings related to the example I'm modifing. 
Do you agree with that?

Comment (jurgob) [reply Cargo.toml:24]: while I was waiting, I've also prepared a branch where I've fixed all the warnings in this project: https://github.com/jurgob/examples/tree/fix_warnings. 

I would do a PR with that after this one is merged. 

(in this PR I've also tested manually my changes, in that branch I did not, but the change is quite the same in all the branch). 


If you prefer I can merge that branch in this PR so there's gonna be 0 warnings

let me know @k0nserv

Comment (jurgob) [reply Cargo.toml]: anyway, I see your concern there. 
I've changed that importing only the codec feature

Comment (jurgob) [reply Cargo.toml]: I've limited the warning fix to this example, but I've fixed that error to make the pass the ci checks

Comment (k0nserv) [reply Cargo.toml]: Hmm I just noticed that it was already broken on `main` which means CI failing for this PR is probably a red herring anyway

Comment (jurgob) [reply Cargo.toml]: so how do you suggest I proceed @k0nserv ?

in the meantime I've fixed the CI problem in another PR: https://github.com/webrtc-rs/examples/pull/14

the problem is that PR is including this PR (which is really 1 readme line change and 3 lines of code). 

can we close this and we just move on the next pr?

Comment (jurgob) [reply Cargo.toml]: as an alternative, I can remove the example change that I did from the pr #14 , and once that is merged we can continue doing this PR.

File: examples/play-from-disk-vpx/play-from-disk-vpx.rs [hunk 2]
```diff
@@ -28,12 +28,13 @@ const OGG_PAGE_DURATION: Duration = Duration::from_millis(20);
...
 #[tokio::main]
 async fn main() -> Result<()> {
-    let mut app = App::new("play-from-disk-vpx")
+    let mut app = Command::new("play-from-disk-vpx")
         .version("0.1.0")
         .author("Rain Liu <yliu@webrtc.rs>")
         .about("An example of play-from-disk-vpx.")
         .setting(AppSettings::DeriveDisplayOrder)
-        .setting(AppSettings::SubcommandsNegateReqs)
+        // .setting(AppSettings::SubcommandsNegateReqs)
```
Comment (k0nserv) [inline examples/play-from-disk-vpx/play-from-disk-vpx.rs:36]: Can we remove this line too?

**Review by k0nserv — APPROVED** [review]

File: examples/play-from-disk-vpx/play-from-disk-vpx.rs (hunk 2 above)
Comment (jurgob) [reply examples/play-from-disk-vpx/play-from-disk-vpx.rs]: done
